proto-docs/status.md

```diff
@@ -0,0 +1,138 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [status/types.proto](#status/types.proto)
+
+  - Messages
+    - [Status](#neo.fs.v2.status.Status)
+    - [Status.Detail](#neo.fs.v2.status.Status.Detail)
+    
+
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="status/types.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## status/types.proto
+
+
+ <!-- end services -->
+
+
+<a name="neo.fs.v2.status.Status"></a>
+
+### Message Status
+Declares the general format of the status returns of the NeoFS RPC protocol.
+Status is present in all response messages. Each RPC of NeoFS protocol
+describes the possible outcomes and details of the operation.
+
+Enumerated with code. Any unique result of an operation in NeoFS is
```

Comment:
What is enumerated with code? I can guess but it doesn't sound clear enough.

Reply:
Paraphrased.